src/lib/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ find_package(ZLIB REQUIRED)
 # required packages
 find_package(JSON-C 0.11 REQUIRED)
 if (CRYPTO_BACKEND_BOTAN3)
-  find_package(Botan 3.0.0 REQUIRED)
+  find_package(Botan 3.2.0 REQUIRED)
```

Comment:
@TJ-91 This should be done only if PQC is enabled, for non-PQC builds we need to support Botan 3.0.0 and 3.1.1 as well.

Reply:
@ni4 ah, I see. I hope it's good now